### ISSUE TITLE
[TSDoc> Search] - Ensures every ARIA button, link and menu item has an accessible name for the hamburger icon

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#240]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/
Tool: Accessibility Insights for Web

**Repro Steps:**
1. Open the above URL in edge dev browser and login with v-.
2. Run the fast pass of accessibility insights for web.
3. Observe the issue.

**Actual Result:**
Ensures every ARIA button, link and menu item does not have an accessible name for the hamburger icon.

**Expected Result:**
Ensures every ARIA button, link and menu item should have an accessible name for the hamburger icon.

**Element path:**
.waffle_FRnN

**Snippet:**
<div role="button" class="waffle_FRnN waffleInteraction_uHqh"><img src="/images/suitenav/rs-waffle.svg"></div>

**How to fix:** 
Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

**User Impact:**
It will impact the screen reader user as they will not be able to get to know the name of control also, it will impact the voice access user as they will not be able to access the element using voice command.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

![Ensures every ARIA button, link and menuitem has an accessible name for the Hamburger button](https://github.com/microsoft/rushstack-websites/assets/95913748/8e578da5-4cca-4a7e-b660-40523e170b1a)

![Ensures every ARIA button, link and menuitem has an accessible name for the hamburger icon](https://github.com/microsoft/rushstack-websites/assets/95913748/a2409e3c-97b2-424e-9116-45538d53fea0)
